### PR TITLE
Update download-sql-server-data-tools-ssdt.md

### PR DESCRIPTION
**Pull request by jasonhorner**
As of VS version v17.7 Serverless pools are supported: https://learn.microsoft.com/en-us/visualstudio/releases/2022/release-notes-v17.7#support-for-serverless-sql-pool-in-ssdt

### CODE REVIEW

File: docs/ssdt/download-sql-server-data-tools-ssdt.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ For Analysis Services, Integration Services, or Reporting Services projects, you
 
 | Project Templates | SQL Platforms Supported |
 | --- | --- |
-| Relational databases | [!INCLUDE [sssql16-md](../includes/sssql16-md.md)] - [!INCLUDE [sssql22-md](../includes/sssql22-md.md)]<br /><br />Azure SQL Database, Azure SQL Managed Instance<br /><br />Azure Synapse Analytics (dedicated pools only) |
+| Relational databases | [!INCLUDE [sssql16-md](../includes/sssql16-md.md)] - [!INCLUDE [sssql22-md](../includes/sssql22-md.md)]<br /><br />Azure SQL Database, Azure SQL Managed Instance<br /><br />Azure Synapse Analytics Dedicated Pools <br /> Azure Synapse Analytics Serverless Pools (requires VS2022 17.7 see [release Notes](https://learn.microsoft.com/en-us/visualstudio/releases/2022/release-notes-v17.7#support-for-serverless-sql-pool-in-ssdt)) |
```
Comment (dzsquared) [inline docs/ssdt/download-sql-server-data-tools-ssdt.md]: looks great, thank you for this!
once you remove the en-us from `https://learn.microsoft.com/en-us/visualstudio/releases/2022/release-notes-v17.7#support-for-serverless-sql-pool-in-ssdt` I can sign off.

the url https://learn.microsoft.com/visualstudio/releases/2022/release-notes-v17.7#support-for-serverless-sql-pool-in-ssdt will auto-resolve to a user's docs language

**Review by dzsquared — CHANGES_REQUESTED** [review]

File: docs/ssdt/download-sql-server-data-tools-ssdt.md (hunk 1 above)
Comment (Court72) [inline docs/ssdt/download-sql-server-data-tools-ssdt.md]:
```suggestion
| Relational databases | [!INCLUDE [sssql16-md](../includes/sssql16-md.md)] - [!INCLUDE [sssql22-md](../includes/sssql22-md.md)]<br /><br />Azure SQL Database, Azure SQL Managed Instance<br /><br />Azure Synapse Analytics Dedicated Pools <br /> Azure Synapse Analytics Serverless Pools (requires VS2022 17.7 see [release Notes](/visualstudio/releases/2022/release-notes-v17.7#support-for-serverless-sql-pool-in-ssdt)) |
```

**Comment by Court72** [comment]
Thanks, @dzsquared.

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by dzsquared** [comment]
#sign-off

**Review by dzsquared — APPROVED** [review]

**Pull request merged**